### PR TITLE
COM-2006 and COM-2005 See you soon page and Let us know when you arri…

### PR DESCRIPTION
**Pull request by hrishid**
Added two templates regarding COM-2006 Curbside-seeyousoon and COM-2005 Curbside-Arrive in back-office.

### CODE REVIEW

File: templates/back-office/crubside-arrive-content.hypr [hunk 1]
```diff
@@ -0,0 +1,42 @@
+<div class="curbside-page-content">
+    <div>
+        <p>{{labels.curbsideSeeYouSoon}}</p>
+        <p>{{ labels.curbsideShipmentText }} </p>
+    </div>
+
+    <div class="curbside-spacing">
+        <button class="mz-delivery-button mz-button mz-button-large" onclick="location.href = 'http://{{domainName}}/anonymous-notification/curbsideInfo/{{model.shipmentNumber}}/{{model.orderId}}';"> {{ labels.CurbSidebtnText }}  </button>
```
Comment (Amjcraft) [inline templates/back-office/crubside-arrive-content.hypr:8]: Don't hard code protocol. Use '//' instead.

File: templates/back-office/crubside-arrive-content.hypr [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+        <ul>
+        <li>{{labels.curbsideIdText}}</li>
+        <li>{{labels.curbsideRefundText}}</li>
+        <li>{{labels.curbsideLandingText}}</li>
+        </ul>
+    </div>
+
+    <div class="curbside-order-details">
+        <div class="curbside-order-details-left">
+         <h2  style="color:white"> <b>{{labels.orderDetails}}</b></h2>
```
Comment (Amjcraft) [inline templates/back-office/crubside-arrive-content.hypr:23]: Other than perhaps emails please do not use inlines styles. 

Our customers heavily customize these templates and it is easier to not have to find and remove inline styles.

**Review by Amjcraft — CHANGES_REQUESTED** [review]
A few small things.

Please remove inline styles

**Review by Amjcraft — CHANGES_REQUESTED** [review]
Few changes

File: theme.json [hunk 3]
```diff
@@ -280,6 +280,18 @@
       "title": "Return Receipt",
       "template": "back-office/return-receipt",
       "customEditor": "back-office@mozu"
+    },
+    {
+      "id": "crubside-arrive",
```
Comment (ghost) [inline theme.json:285]: Fix this spelling error. Should be `curbside-arrive` with title `Curbside Arrive`. Template should be `back-office/curbside-arrive`.

File: theme.json [hunk 4]
```diff
@@ -280,6 +280,18 @@
...
+      "title": "Crubside Arrive",
+      "template": "back-office/crubside-arrive",
+      "customEditor": "back-office@mozu"
+    },
+    {
+      "id": "crubside-seeyousoon",
```
Comment (ghost) [inline theme.json:291]: Fix this spelling error. Should be `curbside-seeyousoon` with title `Curbside See You Soon`. Template should be `back-office/curbside-seeyousoon`. Let's be consistent in our naming.

File: templates/back-office/crubside-arrive.hypr [hunk 5]
```diff
@@ -0,0 +1,4 @@
+{% extends "back-office/curbside-base" %}
+{% block body-content %}
+{% include "back-office/crubside-arrive-content" %}
```
Comment (ghost) [inline templates/back-office/crubside-arrive.hypr:3]: Spelling error. Rename any misspelled templates.

File: templates/back-office/crubside-arrive-content.hypr (hunk 2 above)
Comment (ghost) [reply templates/back-office/crubside-arrive-content.hypr:23]: This inline style wasn't removed. Let's remove it.

File: labels/de-DE.json [hunk 6]
```diff
@@ -80,6 +80,16 @@
...
+    "curbsideRefundText": "Si votre colis n'est pas récupéré dans les 5 jours ouvrables, votre commande sera annulée et vous serez remboursé",
+    "curbsideLandingText": "Lorsque vous arrivez au magasin, veuillez cliquer sur le bouton I'm CurbSide illustré ci-dessus. De là, vous serez redirigé vers une page de destination où vous pourrez fournir vos informations de livraison",
+    "curbsideShipmentText": "Votre commande est disponible et prête à être récupérée. Veuillez cliquer sur le bouton I am Curbside ci-dessous pour informer le magasin que vous êtes arrivé. Vous serez dirigé vers une page de destination où vous pourrez informer le magasin où vous êtes garé et comment vous souhaitez que vos articles soient livrés",
+    "curbsideNoteTitle": "Notez s'il vous plaît...",
+    "curbSidebtnText": "I'm CurbSide",
+    "curbsideStoreDetailsTitleText": "Détails du magasin",
+    "CurbsideSeeYouSoonText": "Merci de nous avoir informés que vous êtes en route! vous recevrez un e-mail sous peu avec une procédure à suivre une fois arrivé au magasin.",
+    "curbsidWHenYouArriveText": "À votre arrivée au magasin, veuillez vous enregistrer via le lien fourni dans votre prochain e-mail.",
```
Comment (ghost) [inline labels/de-DE.json:91]: Lowercase 'c' on `CurbsideSeeYouSoonText`. Be consistent.
Fix spelling error on `curbsidWHenYouArriveText` to be `curbsideWhenYouArriveText`.

File: labels/de-DE.json [hunk 7]
```diff
@@ -80,6 +80,16 @@
     "creditDate"   : "Kredit Datum",
     "creditLimit"    : "Kreditlimit",
     "customerAlreadyExists"   : "Ein Kunde ist bereits als {0} registriert. <a href = \"",
+    "curbsideIdText": "Veuillez apporter votre identifiant correspondant au nom de votre commande (sauf indication contraire).",
+    "curbsideRefundText": "Si votre colis n'est pas récupéré dans les 5 jours ouvrables, votre commande sera annulée et vous serez remboursé",
+    "curbsideLandingText": "Lorsque vous arrivez au magasin, veuillez cliquer sur le bouton I'm CurbSide illustré ci-dessus. De là, vous serez redirigé vers une page de destination où vous pourrez fournir vos informations de livraison",
+    "curbsideShipmentText": "Votre commande est disponible et prête à être récupérée. Veuillez cliquer sur le bouton I am Curbside ci-dessous pour informer le magasin que vous êtes arrivé. Vous serez dirigé vers une page de destination où vous pourrez informer le magasin où vous êtes garé et comment vous souhaitez que vos articles soient livrés",
+    "curbsideNoteTitle": "Notez s'il vous plaît...",
+    "curbSidebtnText": "I'm CurbSide",
```
Comment (ghost) [inline labels/de-DE.json:88]: Lowercase 's' on `curbSidebtnText`.

File: templates/back-office/crubside-arrive-content.hypr (hunk 1 above)
Comment (ghost) [reply templates/back-office/crubside-arrive-content.hypr:8]: If this wasn't changed let's do it.

**Review by ghost — COMMENTED** [review]
I know this was already merged and some of it likely fixed by Aaron but ;let's make the changes we both requested and submit a new PR please. Thanks.

**Pull request merged**